### PR TITLE
fix the license

### PR DESCRIPTION
**Pull request by nunogit**

### CODE REVIEW

**Comment by egonw** [comment]
@nunogit, should I wait for the missing JavaDoc?

**Comment by nunogit** [comment]
Please accept, I'll do another pull request later.
Is there a style book/reference for the project?

**Comment by egonw** [comment]
No, we don't. But here goes:

- write JavaDoc
- write unit tests

That's at this moment the most important parts. I have added both when I started with the project, because there was (and is) very little.

File: org.bridgedb.server/src/org/bridgedb/server/Batch.java [hunk 1]
```diff
@@ -66,6 +73,10 @@ public String accept(Representation entity) {
 		return result;
     }
     
+	/**
+	 * @param entity
+	 * @return
+	 */
```
Comment (egonw) [inline org.bridgedb.server/src/org/bridgedb/server/Batch.java]: Okay, just for the record: templates do not qualify as JavaDoc to me :)

**Pull request merged**